dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1490,8 +1490,8 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 		return;
 	}
 
-	//TESTING IMMUNITY
-	if (chatCommand == "immune" && args.size() >= 1 && entity->GetGMLevel() >= eGameMasterLevel::DEVELOPER) {
+	//Testing basic attack immunity
+	if (chatCommand == "attackimmune" && args.size() >= 1 && entity->GetGMLevel() >= eGameMasterLevel::DEVELOPER) {
```

Comment:
since we'll be keeping this in, don't forget to add information about the command to the [docs here](https://github.com/DarkflameUniverse/DarkflameServer/blob/main/docs/Commands.md)